test/tests/functional/pbs_qstat_dsv.py

```diff
@@ -55,15 +55,28 @@ def parse_dsv(self, attrs_qstatf, jid, qstat_type, delimiter=None):
             delim = " "
         if qstat_type == "job":
             cmd = ' -f -F dsv ' + delim + " " + str(jid)
+            qstat_cmd_dsv = os.path.join(self.server.pbs_conf['PBS_EXEC'],
+                                         'bin', 'qstat') + cmd
             qstat_cmd = os.path.join(self.server.pbs_conf['PBS_EXEC'],
-                                     'bin', 'qstat') + cmd
+                                     'bin', 'qstat') + ' -f ' + str(jid)
```

Comment:
If i do PTL status call and take the keys , PTL formats 'Job Id: <job id>' into 'id' . If i send that list , test will fail as id is missing . I think the current approach is better , as it tests the real command outputs .